src/pages/index.tsx

```diff
@@ -43,7 +44,16 @@ const IndexPage = () => (
       <meta content="#ff7f0e" data-react-helmet="true" name="theme-color" />
     </Head>
     <h1>hello from page index</h1>
+
+    <Wrapper>
```

Comment:
I think the Wrapper should be at the "_app.tsx" level as it's global throughout all the routes.
https://nextjs.org/docs/advanced-features/custom-app